public/apps/configuration/constants.tsx

```diff
@@ -61,6 +61,11 @@ export const CLUSTER_PERMISSIONS: string[] = [
   'cluster:admin/opendistro/alerting/monitor/get',
   'cluster:admin/opendistro/alerting/monitor/search',
   'cluster:admin/opendistro/alerting/monitor/write',
+  'cluster:admin/opendistro/notebooks/create',
+  'cluster:admin/opendistro/notebooks/update',
+  'cluster:admin/opendistro/notebooks/delete',
```

Comment:
Arrangement of all the added permissions should be in alphabetical order to maintain consistency.